components/brave_ads/browser/DEPS

```diff
@@ -4,67 +4,4 @@ include_rules = [
   "-bat/ads/internal",
   "+bat/ledger/mojom_structs.h",
```

Comment:
can you also check what's left here. maybe some of these are not needed anymore?